### PR TITLE
New Guide: Using Visual Studio Code for Theme Development

### PR DESCRIPTION
**Pull request by damiankorcz**
## Edited
- Edited the "damiankorcz" note - Added this Guide under a new Guides heading, Added a new alias for my username and updated the Sponsor this Author section.
- Edited the "for Theme Designers" note - Added the "Using Visual Studio Code for Theme Development" Guide into the list.

## Added
- Added the "Using Visual Studio Code for Theme Development.md" Guide to the "Guides" folder.
- Added attachments for the "Using Visual Studio Code for Theme Development" note into the "02 Attachments" folder.

## Checklist
- [x] before creating a new note, I searched the vault
- [x] new notes have the `.md` extension
- [x] (if applicable) attached images have descriptive file names
- [x] (if applicable) for new notes in the folder "04 - Guides, Workflows, & Courses", I added a link to them in one of the "for {group X}" overviews: https://publish.obsidian.md/hub/04+-+Guides%2C+Workflows%2C+%26+Courses/%F0%9F%97%82%EF%B8%8F+04+-+Guides%2C+Workflows%2C+%26+Courses

### CODE REVIEW

File: 04 - Guides, Workflows, & Courses/Guides/Using Visual Studio Code for Theme Development.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+Here is a list of useful Extensions:
+
+### General
+- [**Color Highlight**](https://marketplace.visualstudio.com/items?itemName=naumovs.color-highlight) - Adds a background of the color corresponding to the color value in the code. 
+- [**Code Spell Checker**](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) - Spell checking for VSCode. By Default it's set to US English and comes with GB English as well. Other languages are available as additional Extensions.
+
+### Sass/SCSS
+- [**Sass**](https://marketplace.visualstudio.com/items?itemName=Syler.sass-indented) - Indented Sass syntax highlighting, autocomplete & Formatter for VSCode.
+- [**SCSS IntelliSense**](https://marketplace.visualstudio.com/items?itemName=mrmlnc.vscode-scss) - SCSS IntelliSense (Variables, Mixins and Functions) for all files in the workspace.
+- [**Stylelint**](https://marketplace.visualstudio.com/items?itemName=stylelint.vscode-stylelint) - A mighty, modern linter that helps you avoid errors and enforce conventions in your styles.
```
Comment (kometenstaub) [inline 04 - Guides, Workflows, & Courses/Guides/Using Visual Studio Code for Theme Development.md:229]: A link to the stylelint guide could be added here.

File: 04 - Guides, Workflows, & Courses/Guides/Using Visual Studio Code for Theme Development.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
+---
+aliases: 
+- vscode
+- visual studio code
```
Comment (kometenstaub) [inline 04 - Guides, Workflows, & Courses/Guides/Using Visual Studio Code for Theme Development.md:4]: The aliases show that this file could be split up. A lot of it is useful general VSCode knowledge that could live in its own file and be embedded by heading when this guide references it. The more specific CSS references/explanations etc. could be in this file.
That way I would only add `using VSCode for theme development` as alias of this note and `VSCode` for the `Visual studio note`.

If we do it like that, we'll have its own VSCode guide that can be linked to/expanded.

**Review by jdanielmourao — APPROVED** [review]
Everything looks good to me. Also, nice theme recommendation! 😉

File: 04 - Guides, Workflows, & Courses/Guides/Using Visual Studio Code for Theme Development.md [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+
+## 💾 Settings Sync
+If you happen to be working with VSCode across multiple devices it is a good idea to setup Settings Sync. It will synchronise settings, key bindings and installed extensions across your machines. You will need to use a Microsoft account or a GitHub account to sync.
+
+Simply click on the `Cog` icon in the bottom left corner and select `Turn On Settings Sync...`. You will be presented with the options for data to be synchronised. 
+
+**More Info: [Settings Sync](https://code.visualstudio.com/docs/editor/settings-sync)**
+
+## 🔌 Extensions
+Here is a list of useful Extensions:
```
Comment (jdanielmourao) [inline 04 - Guides, Workflows, & Courses/Guides/Using Visual Studio Code for Theme Development.md:220]: Here are a couple of suggestions:
Change Color Format - enables you to quickly convert colors into different color formats
SCSS IntelliSense - useful autocompletion for Sass variables, mixins, etc

**Review by SlRvb — APPROVED** [review]
Looks good to me 👍🏽

**Pull request merged**